### PR TITLE
Add i3 icon

### PR DESCRIPTION
**Pull request by zrfisaac**
i3

**Issue:** #7217

**Similarweb rank:** [203,571](https://i3wm.org/)

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
i3 is a [tiling window manager](https://en.wikipedia.org/wiki/Tiling_window_manager), completely written from scratch. The target platforms are GNU/Linux and BSD operating systems, our code is Free and Open Source Software (FOSS) under the BSD license. i3 is primarily targeted at advanced users and developers.

### CODE REVIEW

File: icons/i3.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...0 0 1.897-.004A1.341 1.341 0 0 0 13.214.39 1.341 1.341 0 0 0 12.265 0zm3.066 3.168a1.342 1.342 0 0 0-.93 2.304l6.219 6.194c.738.735.857 1.618.488 2.897-.37 1.28-1.325 2.78-2.553 4.013-1.227 1.232-2.724 2.194-4.001 2.569-1.278.375-2.162.26-2.9-.476-1.723-1.715-3.26-3.275-4.379-4.402-.56-.564-1.012-1.019-1.339-1.339-.163-.16-.292-.283-.403-.385-.056-.05-.102-.092-.18-.152-.039-.03-.075-.065-.195-.132-.06-.034-.133-.082-.337-.134a1.314 1.314 0 0 0-1.283.354 1.342 1.342 0 0 0-.394 1.122c.051.415.364.73.653.994.066.062.142.132.262.25.305.298.755.75 1.312 1.312 1.114 1.122 2.657 2.687 4.39 4.413 1.507 1.501 3.667 1.7 5.548 1.149 1.88-.552 3.66-1.757 5.147-3.25 1.487-1.494 2.685-3.278 3.229-5.161s.335-4.042-1.172-5.543l-6.22-6.193c-.253-.26-.6-.405-.963-.404zM6.744 5.544a1.341 1.341 0 0 0-.948.395A1.341 1.341 0 0 0 5.8 7.836a1.341 1.341 0 0 0 1.897-.004 1.341 1.341 0 0 0-.004-1.897 1.341 1.341 0 0 0-.95-.391zm3.151 3.139a1.341 1.341 0 0 0-.93 2.305l6.973 6.943a1.341 1.341 0 1 0 1.893-1.9l-6.973-6.944c-.253-.26-.6-.405-.963-.404zM1.381 10.93a1.341 1.341 0 0 0-.948.395 1.341 1.341 0 0 0 .004 1.897 1.341 1.341 0 0 0 1.897-.004 1.341 1.341 0 0 0-.004-1.897 1.341 1.341 0 0 0-.95-.391z"/></svg>
```
Comment (sisp) [inline icons/i3.svg:1]: The SVG looks good visually, there's no visual difference to the original icon. However, when I extract the icon from the original one and optimize it with SVGOMG, I get ~20% shorter path data.

```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>i3</title><path d="M13.215.39a1.341 1.341 0 1 1-1.893 1.902A1.341 1.341 0 0 1 13.215.39ZM7.694 5.936a1.341 1.341 0 1 1-1.893 1.9 1.341 1.341 0 0 1 1.893-1.9ZM2.33 11.321a1.341 1.341 0 1 1-1.893 1.9 1.341 1.341 0 0 1 1.893-1.9Zm7.566-2.638c.362 0 .71.145.963.404l6.972 6.944a1.341 1.341 0 1 1-1.893 1.9l-6.972-6.943a1.341 1.341 0 0 1 .93-2.305Zm5.435-5.515c.362-.001.71.145.963.404l6.219 6.193c1.507 1.501 1.716 3.66 1.172 5.543-.543 1.883-1.741 3.667-3.228 5.16-1.488 1.494-3.267 2.7-5.148 3.251-1.88.552-4.04.352-5.548-1.15-1.732-1.724-3.276-3.29-4.39-4.412a150.903 150.903 0 0 0-1.311-1.312c-.12-.118-.196-.188-.262-.25-.29-.265-.602-.579-.653-.993-.05-.415.095-.83.393-1.122a1.314 1.314 0 0 1 1.284-.355c.204.052.276.1.336.134.12.067.157.102.196.132.078.06.123.102.179.153.111.1.24.224.404.384.327.32.78.775 1.338 1.339 1.119 1.127 2.657 2.687 4.38 4.402.738.735 1.621.85 2.899.476 1.278-.375 2.774-1.337 4.002-2.569 1.227-1.232 2.183-2.733 2.552-4.012.37-1.28.25-2.163-.488-2.898L14.4 5.472a1.341 1.341 0 0 1 .93-2.304Z"/></svg>

**Review by sisp — CHANGES_REQUESTED** [review]

File: icons/i3.svg (hunk 1 above)
Comment (sachinraja) [reply icons/i3.svg:1]: Nice! @zrfisaac could you switch to this more optimized version?

**Comment by Jivanam** [comment]
Hi [zrfisaac](https://github.com/zrfisaac), could you please accept the suggested changes? Thanks

**Comment by zrfisaac** [comment]
@Jivanam I will fix this and post! 🙌

**Review by PratyushJayachandran — APPROVED** [review]
LGTM, thank you @zrfisaac :tada:

**Pull request merged**